rllib/evaluation/tests/test_agent_collector.py

```diff
@@ -332,6 +332,19 @@ def test_view_requirement_with_shfit_step(self):
         # check if the padding in the beginning is correct
         check(sample_batch["prev_obses"][0], np.ones((3, 1)) * expected_obses[0])
 
+    def test_begin_with_inference(self):
```

Comment:
I tried computing actions from a policy right after init.
This currently results in spawning connectors if they have not been initialized.
VR connector has limited buffer content and would fail within within `_to_float_np_array(d)` (related change further down) because d would be in buffers because we called with the same column before, but empty. This test covers this issue.